cclib/parser/orcaparser.py

```diff
@@ -1384,6 +1388,149 @@ def energy_intensity(line):
+                    etsyms.append(self.mdci_et_mult)
+                sec = []
+                # Header line.
+                # There is no header for CIS.
+                if not cis:
+                    line = next(inputfile)
+                # First orbital contribution line.
+                line = next(inputfile)
+                while "->" in line:
+                    coeff_split = line.split()
+                    # CIS prints coefficients after the orbitals, other modules are reversed.
+                    if not cis:
+                        contrib = coeff_split[0]
+                        start = coeff_split[1]
+                        end = coeff_split[3]
+                    else:
+                        # CIS looks like this:
+                        # 32 ->  37    0.037502 (-0.193653)
+                        # 34 ->  35    0.935796 ( 0.967365)
+                        # 34 ->  36    0.014167 (-0.119024)
+                        # Note that sometimes the coefficient has whitespace between the brackets, sometimes not.
+                        start = coeff_split[0]
+                        end = coeff_split[2]
+                        contrib = "".join(coeff_split[4:])[1:-1]
+                    
+                    try:
+                        # TODO: Unrestricted?
+                        sec.append([(int(start), 0), (int(end), 0), float(contrib)])
+                        
+                    except ValueError:
+                        # Sometimes we come across lines like:
+                        # 0.690372    69 -> x
+                        # Ignore these for now.
+                        pass
+                    
+                    line = next(inputfile)
+                
+                # Sort contributions so largest is first.
+                etsecs.append(sorted(sec, key = lambda sec_item: sec_item[2] **2, reverse = True))
+                
+                if "Ground state amplitude" in line:
+                    # Data currently not parsed. Just skip.
+                    line = self.next_filled_line(inputfile)
+                
+                if "Percentage singles character" in line:
+                    # Data currently not parsed. Just skip.
+                    line = self.next_filled_line(inputfile)
+                    
+                # (Possibly) blank line
+                if "IROOT=" in line:
+                    continue
+                elif line.strip() == "":
+                    line = self.next_filled_line(inputfile)
+                
+                if "Percentage Active Character " in line:
+                    # Data currently not parsed. Just skip.
+                    line = self.next_filled_line(inputfile)
+                    
+                if "Warning:: the state may have not converged with respect to active space" in line:
+                    # Skip this line and the next (which both contain warnings).
+                    # TODO: Capture this warning?
```

Comment:
You can use `self.logger.warning` (the warning method on our instance of https://docs.python.org/3/library/logging.html) to report it.